sdk/storage/storage-blob/src/Pipeline.ts

```diff
@@ -202,8 +202,8 @@ export function newPipeline(
     telemetryPolicy,
     generateClientRequestIdPolicy(),
     new StorageBrowserPolicyFactory(),
-    deserializationPolicy(), // Default deserializationPolicy is provided by protocol layer
     new StorageRetryPolicyFactory(pipelineOptions.retryOptions),
+    deserializationPolicy(), // Default deserializationPolicy is provided by protocol layer
```

Comment:
Should we add a comment here denoting the reason the deserialization policy being placed after the retry policy?

Reply:
Added a comment to `StorageRetryPolicyFactory` line